src/inject/GPMInject/interface/mini.js

```diff
@@ -29,7 +41,13 @@ window.wait(() => {
     //      Same reason as specified in Electron src
     //        --> https://github.com/atom/electron/blob/master/atom/browser/native_window_views.cc
     mainWindow.setMaximumSize(99999999, 999999999);
-    mainWindow.setSize(...oldSize);
+
+    // Save mini size/pos and restore regular size/pos.
+    miniSize = remote.getCurrentWindow().getSize();
```

Comment:
You can get rid of these two lines if we are fetching the size from settings as per my above comment